source/install/self-managed-changelog.md

```diff
@@ -4,12 +4,93 @@
+ - Added thread replies to search results when Collapsed Reply Threads is enabled.
+ - Updated “Terms of Service” terminology to “Terms of Use” product-wide.
+ - Clarified Latex Rendering config setting descriptions and fixed a broken product documentation link.
+ - Updated the “One-click reactions on messages” user setting to “Quick reactions on messages”.
+ - Added tab focus support to the global header and user avatars.
+ - Added a new Replies banner to the right-hand side Thread viewer.
+ - Updated the About Mattermost Team Edition modal to change the community link from `mattermost.org` to `mattermost.com/community/`.
+ - Invite to team modal now auto-focuses its email search input.
+
+#### Enterprise Edition
+ - The **Renew Now** button is no longer shown if the license ID does not exist in the portal.
+ - Added a new dialog for Remove License confirmation.
+ - System Admins are now able to upgrade the server to the Enterprise edition and request the trial license with a single selection to enhance product adoption with an improved user experience.
+
+#### Administration
+ - The config setting ``ServiceSettings.EnableReliableWebSockets`` has promoted to general availability. For compatibility with older clients, the server will always return ``true`` for the ``/v4/config/client`` endpoint.
+ - Added server support for receiving binary (messagepack encoded) websocket messages.
+ - Removed dead struct ``ManifestExecutables``.
+ - Added new flag ``showTeamSidebar`` in ``registerProducts``, which, when set to ``true``, displays the team sidebar in the product.
+ - Memberlist logs and buckets are now parsed by DEBUG, INFO, WARN, or ERROR appropriately.
+ - Increased key length in plugin KV store to 150.
+
+### Bug Fixes
+ - Fixed an issue where when selecting the **Upgrade to Enterprise Edition** option, the upgrade progress bar and the **Restart** button were no longer shown once the progress reached 100%.
+ - Fixed an issue where the user avatar wasn’t removed from the participants list after the user’s only post in a thread was deleted.
+ - Fixed an issue with the exit animation on the invitation modal.
+ - Fixed an issue where the status menu unexpectedly closed when selecting the “Disable Notifications Until” header.
+ - Fixed an issue where using CMD/CTRL + SHIFT + F in global threads did not add a search term automatically.
+ - Fixed the alignment of the “X” button in the “message deleted” system message.
+ - Fixed an issue where the long post “Show More/Less” background was broken in the Thread viewer.
+ - Changed Client4 to properly set Content-Type as application/json on API calls.
+ - Fixed an issue with post hover menu overlap.
+
+### config.json
+
+#### Changes to Team Edition and Enterprise Edition:
+- The config setting ``ServiceSettings.EnableReliableWebSockets`` was removed.
```

Comment:
On the previous section it read it was promoted to GA. But here we say the setting is removed?

Reply:
```suggestion
- The config setting ``ServiceSettings.EnableReliableWebSockets`` was removed, and the ability to buffer messages during a connection loss has been promoted to general availability This setting is enabled for older clients to maintain backwards compatibility.
```

Reply:
> On the previous section it read it was promoted to GA. But here we say the setting is removed?

Basically the same thing. It was a config param, because we allowed users the ability to turn it off in case there was a bug. Now after several releases of building confidence, we have removed the param, and turned it on for everyone.

Strictly speaking - I don't think we ever called the feature "beta". But I'll leave it to you all to word it as you feel.